ext/sockets/tests/socket_import_stream-3.phpt

```diff
@@ -6,9 +6,9 @@ if (!extension_loaded('sockets')) {
 	die('SKIP sockets extension not available.');
 }
 $s = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
-$br = socket_bind($s, '0.0.0.0', 58381);
+$br = socket_bind($s, '0.0.0.0', 58379);
```

Comment:
Would it make sense to declar CONFLICTS for this tests based on the used port number?
That way you dont need to check the whole codebase which tests use which port

Reply:
It's better to adjust the test (if it's very simple, like here), as it allows it to run in parallel. Adding a port-based CONFLICT would prevent spurious failures, but reduce parallelization opportunities.

In the future we might want to switch code to use dynamic port assignment instead (this is what HHVM did in their tests), but that's a lot more effort than changing a fixed port :)